deepcell/applications/multiplex_segmentation.py

```diff
@@ -40,6 +40,75 @@
 from deepcell.model_zoo import PanopticNet
 
 
+import numpy as np
+
+
+def deep_watershed_subcellular(outputs, compartment='cell', min_distance=10, maxima_threshold=0.1,
+                               cell_threshold=0.1, exclude_border=False,
+                               small_objects_threshold=0):
+    """Postprocess model output to generate predictions for distinct cellular compartments
+
+    Args:
+        outputs (list): DeepWatershed model output. A list of
```

Comment:
Can we assume that the order of the compartments in the list is always the same? If so, let's make a note of the appropriate order in the docs